src/vend-number.js

```diff
@@ -27,153 +29,166 @@ const VendNumber = {
+ *        The value to round to the provided number decimal points or two.
+ *
+ * @param [decimalPoints=2]
+ *        The number of decimal points to round the passed value to, or two.
+ *
+ * @return {String} The rounded value.
+ */
+VendNumber.round = function (value, decimalPoints) {
+  // Convert to VendNumber if not already.
+  value = (value instanceof BigNumber) ? value : new VendNumber(value)
 
-    return value.toFixed(decimalPoints)
-  },
+  // 2dp by default.
+  decimalPoints = (typeof decimalPoints === 'number') ? decimalPoints : 2
 
-  /**
-   * Adds a list of values.
-   *
-   * @method add
-   */
-  add: function () {
-    var values = Array.prototype.slice.call(arguments, 0)
-    return VendNumber._executeOperation('plus', values)
-  },
+  return value.toFixed(decimalPoints)
+}
 
-  /**
-   * Subtracts a list of values.
-   *
-   * @method subtract
-   */
-  subtract: function () {
-    var values = Array.prototype.slice.call(arguments, 0)
-    return VendNumber._executeOperation('minus', values)
-  },
+/**
+ * Adds a list of values.
+ *
+ * @method add
+ * @static
+ */
+VendNumber.add = function (...values) {
+  return _executeOperation('plus', values)
+}
 
-  /**
-   * Multiplies a list of values.
-   *
-   * @method multiply
-   */
-  multiply: function () {
-    var values = Array.prototype.slice.call(arguments, 0)
-    return VendNumber._executeOperation('times', values)
-  },
+/**
+ * Subtracts a list of values.
+ *
+ * @method subtract
+ * @static
+ */
+VendNumber.subtract = function (...values) {
+  return _executeOperation('minus', values)
+}
 
-  /**
-   * Divides a list of values.
+/**
+ * Multiplies a list of values.
+ *
+ * @method multiply
+ * @static
+ */
+VendNumber.multiply = function (...values) {
+  return _executeOperation('times', values)
+}
+
+/**
+ * Divides a list of values.
+ *
+ * @method divide
+ * @static
+ */
+VendNumber.divide = function (...values) {
+  return _executeOperation('dividedBy', values)
+}
+
+/**
+ * Executes VendNumber calculation operation on an array of values.
+ *
+ * @private
+ * @method _executeOperation
+ *
+ * @param operation {String}
+ *        The operation to perform on the VendNumber.
+ *
+ * @param values {Array}
+ *        A list of values to perform the operation on (any length).
+ *
+ * @return {Number} The final result or 0 if invalid.
+ */
+function _executeOperation (operation, values) {
+  /*
+   * Executes throwing a VendNumber TypeError, when any method has received an invalid value.
    *
-   * @method divide
+   * @private
+   * @method _displayValueError
    */
-  divide: function () {
-    var values = Array.prototype.slice.call(arguments, 0)
-    return VendNumber._executeOperation('dividedBy', values)
-  },
+  function _displayValueError (value) {
+    throw new TypeError('The VendNumber method must receive a valid String or Number. ' + value)
+  }
 
-  /**
-   * Executes VendNumber calculation operation on an array of values.
+  /*
```

Comment:
AFAIK the original reason this would have been done was because otherwise yuidoc would generate the wrong docs, assuming this method was on the top level class. But I guess seeing as we don't actually generate docs with yuidoc anymore it doesn't matter?